query_optimizer/PhysicalGenerator.cpp

```diff
@@ -108,6 +109,7 @@ P::PhysicalPtr PhysicalGenerator::generateInitialPlan(
 P::PhysicalPtr PhysicalGenerator::optimizePlan() {
   std::vector<std::unique_ptr<Rule<P::Physical>>> rules;
   rules.emplace_back(new PruneColumns());
+  rules.emplace_back(new PushDownLowCostDisjunctivePredicate());
```

Comment:
Could we have some comments regarding the order of `rules`, i.e., why put `PushDownLowCostDisjunctivePredicate` here?

If I understand correctly, `PushDownLowCostDisjunctivePredicate` may generate new `Physical Select`s. Would the rest of `rules` eliminate multiple `Physical Select`s into one?